plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/commands/RefreshAllSchemasHandler.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.ui.editors.sql.commands;
+
+import org.eclipse.core.commands.ExecutionEvent;
+import org.eclipse.core.commands.ExecutionException;
+
+public class RefreshAllSchemasHandler extends AbstractSchemaHandler {
+
+    @Override
+    public Object execute(ExecutionEvent event) throws ExecutionException {
+        var sqlEditor = getEditor(event);
+        if (sqlEditor != null) {
+            var contextDefaults = getExecutionContext(sqlEditor);
+            var catalog = contextDefaults.getDefaultCatalog();
+            if (catalog != null) {
+                try {
+                    var schemaNodes = getSchemaNodes(sqlEditor, catalog);
+                    refreshNodes(schemaNodes);
+                } catch (Exception e) {
+                    e.printStackTrace();
```

Comment:
Never ever use the `printStackTrace`. Either log or re-throw it.

Reply:
Fixed